### DISCUSSION TITLE
Create MTA-STSV1 and DANE record?

### DISCUSSION THREAD

**Discussion by sakearzoo** [anonaddy/docker#148] [Q&A]
With normal self hosting i can create `.well-known` folder where i can place the MTA-STSV1 file but i couldn't understand how to do the same with the docker container. Is there a way creating MTA-STSv1 records and DANE records with anonaddy docker container?

**Comment by eleaner** [comment]
did you figure it out?

**Comment by eleaner** [comment]
I think it does not differ much. 
Selfhosting with docker will still require reverse proxy, so can you use it to expose .well-known too
see here
https://github.com/anonaddy/docker/blob/master/examples/nginx/nginx/templates/mta-sts.conf.template
